pwndbg/commands/context.py

```diff
@@ -204,7 +200,7 @@ def context_backtrace(frame_count=10, with_banner=True):
     result = []
 
     if with_banner:
-        result.append(pwndbg.color.blue(pwndbg.ui.banner("backtrace")))
+        result.append(pwndbg.color.banner(pwndbg.ui.banner("backtrace")))
```

Comment:
sure why not, just don't wanted to refactor too much of how you did things before. will do